app/views/storage_locations/show.html.erb

```diff
@@ -161,3 +165,11 @@
   <li>Save the file as a csv file.</li>
   <li>DO NOT CHANGE ANY ITEM NAME.</li>
 <% end %>
+
+<script>
+  $(document).ready(() => {
```

Comment:
Should this be in application.js?

Reply:
I'm not a fan of bundling functionality that affects just one page in with code that's loaded on every page of the application. Less for performance reasons and more that I see the application JS file as functionality that should be broadly useful across at least a few pages. If it's ultra-specific I'd rather have it on the page itself. Theoretically I could create a separate JS file that's included just by this page, but that also seems unnecessary.

Reply:
100%; that makes sense.